components/Ink/examples/InkTypicalUse.m

```diff
@@ -17,8 +17,7 @@
 #import <UIKit/UIKit.h>
 
 #import "MaterialInk.h"
-
```

Comment:
If possible, I think we want to preserve the space between the import categories.